tests/app/va/va_profile/test_va_profile_client.py

```diff
@@ -1,764 +1,290 @@
 import pytest
```

Comment:
This entire test file was rewritten/reformatted.  Probably best to just view it [on its own](https://github.com/department-of-veterans-affairs/notification-api/blob/dcd8276c632b0ff99c390acc4967ea09e48bf953/tests/app/va/va_profile/test_va_profile_client.py) as opposed to trying to compare changes.